### PR TITLE
Close #LGVISIUM-84: Added possibility to pass the AWS Credentials in an .env file or as environment variables

### PR DESCRIPTION
**Pull request by dcleres**
I implemented the changes required by test users to make it possible to use environment variables to specify the AWS credentials to the API. The credentials can also be specified in an `.env` file now. 

The `create_pngs` path also now directly returns the file path within the S3 bucket without specifying the bucket URL.

### CODE REVIEW

File: tests/test_data_extraction_from_bbox.py [hunk 1]
```diff
@@ -59,6 +59,8 @@ def get_text_request_on_rotated_pdf():
 @pytest.fixture(scope="function")
 def upload_test_pdf(s3_client):
     """Upload a test PDF file to S3."""
+    s3_client.list_buckets()
```
[the remark this answers is not in the document]
Comment (dcleres) [reply tests/test_data_extraction_from_bbox.py:62]: It was a test for me to see more readily if the mocking of the S3 bucket worked. You are right; it is not needed anymore.

File: tests/test_create_pngs.py [hunk 2]
```diff
@@ -49,11 +49,10 @@ def test_create_pngs_success(test_client: TestClient, s3_client, upload_test_pdf
 
     # Verify that PNG files are uploaded to S3
     for png_url in json_response["png_urls"]:
```
[the remark this answers is not in the document]
Comment (dcleres) [reply tests/test_create_pngs.py:51]: I opened a Ticket on Jira for this.

File: README.md [hunk 3]
```diff
@@ -334,21 +335,84 @@ The borehole application offers a given amount of functionalities (extract text,
...
     This command will start the container and map port 8000 of the container to port 8000 of the host machine.
 
-5. **Run the docker image with the AWS credentials**
+4.2. **Run the docker image with the AWS credentials**
+
+4.2.1. **Using a `~/.aws` file**
 
 If you have the AWS credentials configured locally in the `~/.aws` file, you can run the following command to forward your AWS credentials to the docker container 
 
+    To run the docker image from `Dockerfile` locally:
```
[the remark this answers is not in the document]
Comment (dcleres) [reply README.md:344]: Good catch thank you!

**Comment by dcleres** [comment]
@stijnvermeeren-swisstopo thank you very much for your review. I addressed your comment and replied to the questions. Regarding the naming of the field that contains the naming and locations of the created PNG files after calling the API, I opened a ticket for that on Jira, as we need to sync with Geowerkstatt.

File: src/app/common/aws.py [hunk 4]
```diff
@@ -58,21 +94,23 @@ def load_data_from_aws(filename: Path, prefix: str = "") -> bytes:
     """Load a document from AWS S3.
 
     Args:
-        filename (str): The filename of the PNG image.
+        filename (str): The filename of the PDF image.
```
Comment (stijnvermeeren-swisstopo) [inline src/app/common/aws.py:97]:
```suggestion
        filename (str): The filename of the document.
```

**Review by stijnvermeeren-swisstopo — CHANGES_REQUESTED** [review]
I don't see the requested changes in the README.md yet, did you by any change forget to push them?

**Comment by dcleres** [comment]
@stijnvermeeren-swisstopo Yes, indeed, the README was not included in my commit. I also addressed the comment you made. Thank you for your patience.

**Review by stijnvermeeren-swisstopo — APPROVED** [review]
LGTM!

**Pull request merged**